src/modules/75-cf/pages/feature-flags/FlagResult.tsx

```diff
@@ -12,22 +12,26 @@ import { Container, Heading, Layout, Text, Utils } from '@harness/uicore'
 import { Color } from '@harness/design-system'
 import type { ContainerProps } from '@harness/uicore/dist/components/Container/Container'
 import { useStrings } from 'framework/strings'
-import type { Feature } from 'services/cf'
+import type { Feature, Results } from 'services/cf'
 import { VariationWithIcon } from '@cf/components/VariationWithIcon/VariationWithIcon'
 import { formatNumber } from '@cf/utils/CFUtils'
 import css from './FeatureFlagsPage.module.scss'
 
 export interface FlagResultProps extends ContainerProps {
   feature: Feature
+  metrics: Results[] | undefined
```

Comment:
Could this just be marked as optional?
```suggestion
  metrics?: Results[]
```

Reply:
Yep - will fix as part of the unit test ticket coming next